src/services/role/hooks/userRoles.js

```diff
@@ -1,7 +1,7 @@
 const { Forbidden } = require('@feathersjs/errors');
 
 exports.restrictGetToCurrentUser = (context) => {
-	if (context.params.account.userId.toString() !== context.id) {
+	if (context.params.account.userId.toString() !== context.id.toString()) {
```

Comment:
we have object id comparator utils in https://github.com/schul-cloud/schulcloud-server/blob/develop/src/helper/compare.js#L14 which is also checking for isValidObjectId which. is missing here.
assume both would be a empty string, this would still seamlessly succeed. you may use it or instead manually check that one of the parameters is a valid id